client/package.json

```diff
@@ -29,6 +29,8 @@
     "core-js": "^2.4.1",
     "font-awesome": "^4.7.0",
     "jwt-decode": "^2.2.0",
+    "moment": "^2.22.2",
+    "ngx-moment": "^3.1.0",
```

Comment:
Added this as well even it is not used yet, its purpose is to be used as angular pipes with momentjs features